docs/styleguide/templates/master.html

```diff
@@ -43,13 +43,37 @@
 {% block header %}
   {% include "app_header.html" %}
 {% endblock %}
-    <main class="docs-app__main" role="main">
-
+    <main class="docs-app__main container-fluid" role="main">
+        <div class="mobile-left-side-navigation">
```

Comment:
Can this go before `main`? The `main` container should be reserved for page-specific content, all app-level stuff should be outside of it.

And please pull this into an include to keep the master template at a very high-level.